source/code/include/playfab/PlayFabCallRequestContainer.h

```diff
@@ -15,11 +15,13 @@ namespace PlayFab
             const std::unordered_map<std::string, std::string>& headers,
             const std::string& requestBody,
             CallRequestContainerCallback callback,
-            void* customData = nullptr,
-            std::shared_ptr<PlayFabApiSettings> apiSettings = nullptr);
+            std::shared_ptr<PlayFabApiSettings> apiSettings,
```

Comment:
nit: any particular reason these don't need to default to nullptr anymore?

Reply:
1. I changed the order of the parameters
2. I wanted to ensure that every place where this is made, it's made with deliberate intent

What I mean with 2 is that we still pass nullptr sometimes, but we pass it deliberately, not by accident.